crates/holochain/src/conductor/cell.rs

```diff
@@ -873,6 +882,7 @@ impl Cell {
     async fn handle_call_remote(
         &self,
         from_agent: AgentPubKey,
+        zome_call_params_serialized: ExternIO,
         from_signature: Signature,
```

Comment:
Why are the internals and the serialized payload both here? Ideally, we'd only have one or the other right?

Reply:
Hmm, yes, I think you're right. I'm trying that out.

Reply:
Done too. Great you pointed that out, it's prettier now.